Airports/EGKK/SMR/Regions.txt

```diff
@@ -2418,3 +2557,21 @@ N051.09.08.672 W000.11.23.582
 N051.09.08.398 W000.11.25.892
 N051.09.09.383 W000.11.26.286
 N051.09.09.281 W000.11.27.355
+; - Near_S
```

Comment:
As per previous comments, this isn't adding anything of great value to the controller, only what looks like a car park. 

![image](https://user-images.githubusercontent.com/2829581/57585127-b384de80-74db-11e9-850c-858e90c08c51.png)